app/furniture/marketplace/management_component.rb

```diff
@@ -0,0 +1,11 @@
+class Marketplace
+  class ManagementComponent < ApplicationComponent
```

Comment:
My brain keeps wanting this to be the `Marketplace::EditComponent`, but it's not *really* the `Marketplace#edit` action and is a more general purpose layout... I don't think this comment is helpful but it is what I noticed when I was reading.